### DISCUSSION TITLE
当配置完【稍后读支持读取本地缓存】仍不生效的解决办法

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2973] [重要通知]
## 问题描述

> 当按照下面的教程配置后，无法在 `output` 文件夹生成 `id-title` 的时

- https://github.com/Kenshin/simpread/discussions/2146
- https://github.com/Kenshin/simpread/discussions/2220

> 下图为有 `id-title` 的效果

![image](https://user-images.githubusercontent.com/81074/138629993-ea579840-8ed2-4eef-9b15-a1643ec144bc.png)

## 解决方案

> 选项页 → 高级设定 → 稍后读（**必须要开启下面的选项**）

![image](https://user-images.githubusercontent.com/81074/138629445-b05a3127-3d57-4075-adcc-35aab085bf95.png)

> 选项页 → 高级设定 → 标注（**建议开启下面的选项**）

![image](https://user-images.githubusercontent.com/81074/138630288-380a5526-fb14-41af-8dc4-0fe9d3c72092.png)

## 备注

> 已经在 https://github.com/Kenshin/simpread/discussions/2146 增加了上述设置，如果你是 _2021年10月25日_ 之后查看到此的用户，请按照教程配置即可。

***

PS：感谢 @myjeshurun 提供的配置文件，我才能发现这个情况，教程中已增加此提示。

**Comment by myjeshurun** [comment]
哈哈，虽然等了许久，最终问题得到了圆满的解决，非常感谢～

**Reply by Kenshin** [reply]
确实太久了，之前有些事情一直在忙  😂  还好不白等。

🙏
